packages/world-modules/src/modules/erc20-puppet/tables/ERC20Metadata.sol

```diff
@@ -29,28 +26,10 @@ library ERC20Metadata {
   FieldLayout constant _fieldLayout =
```

Comment:
Would be interesting to have a similar `// Hex-encoded ....` comment explaining how the field layout is generated!

Reply:
I considered that but didn't have an obvious way to generate a human-readable comment for it so I left it (it never had one)